### PR TITLE
[DRAFT] New Learn: Cascade layers

### PR DESCRIPTION
**Pull request by estelle**
#### Metadata

- [x] Adds a new document
- [ ] Rewrites (or significantly expands) a document
- [ ] Fixes a typo, bug, or other error

### CODE REVIEW

File: files/en-us/learn/css/building_blocks/cascade_layers/index.md [hunk 1]
```diff
@@ -0,0 +1,235 @@
...
+  - Cascade
+  - Inheritance
+  - Learn
+  - rules
+  - source order
+  - specificity
+---
+{{LearnSidebar}}{{NextMenu("Learn/CSS/Building_blocks/Selectors", "Learn/CSS/Building_blocks")}}
+
+The aim of this lesson is to develop your understanding of some of the most fundamental concepts of CSS — the cascade, specificity, and inheritance — which control how CSS is applied to HTML and how conflicts between style declarations are resolved.
```
Comment (MrBrain295) [inline files/en-us/learn/css/building_blocks/cascade_layers/index.md:16]:
```suggestion
The aim of this lesson is to develop your understanding of some of the most fundamental concepts of CSS—the cascade, specificity, and inheritance—which control how CSS is applied to HTML and how conflicts between style declarations are resolved.
```

File: files/en-us/learn/css/building_blocks/cascade_layers/index.md [hunk 2]
```diff
@@ -0,0 +1,235 @@
...
+      </td>
+    </tr>
+  </tbody>
+</table>
+
+For each property applied to an element, there can only be one value. While the name of this document is a specific color and font size, if you inspect that heading in your browser developer tools “styles” panel, you may notice that there are styles crossed out. The crossed-out styles are the property values that were not applied (even if the value is the same as the current value) due to the cascade. 
+ 
+## Quick recap: the cascade 
+ 
+The cascade, the C in CSS, is the method by which styles cascade together. The user agent goes through several very clearly defined steps to determine which single value gets assigned for every property of every element. We will briefly list the steps, then dig deeper into step 4, cascade layers, which is what you came here to learn:
```
Comment (MrBrain295) [inline files/en-us/learn/css/building_blocks/cascade_layers/index.md:54]:
```suggestion
The cascade, the C in CSS, is the method by which styles cascade together. The user agent goes through several very clearly defined steps to determine which single value gets assigned to every property of every element. We will briefly list the steps, then dig deeper into step 4, cascade layers, which is what you came here to learn:
```

File: files/en-us/learn/css/building_blocks/cascade_layers/index.md (hunk 1 above)
Comment (teoli2003) [reply files/en-us/learn/css/building_blocks/cascade_layers/index.md:16]: The original is better.

Comment (MrBrain295) [reply files/en-us/learn/css/building_blocks/cascade_layers/index.md:16]: https://www.uis.edu/learning-hub/writing-resources/handouts/learning-hub/dashes-and-parentheses

**Comment by estelle** [comment]
Closes https://github.com/openwebdocs/project/issues/94

**Pull request closed without merging**